src/components/Content/Content.tsx

```diff
@@ -0,0 +1,196 @@
+import * as React from "react";
+import { ReactElement, useState, useEffect } from "react";
+import { ContentProps, ContentType } from "./ContentTypes";
+
+// Components
+import Button from "../Button";
+import Chip from "../Chip";
+import {
+	MainWrapper,
+	Title,
+	TitleWrapper,
+	ButtonsWrapper,
+	Label,
+	Value,
+	LabelValueWrapper,
+	Paragraph,
+	ChipsWrapper,
+	LabelWrapper,
+	ContentRow,
+	ContentColumn,
+} from "./Content.styled";
+import EditIcon from "@mui/icons-material/Edit";
+import AddIcon from "@mui/icons-material/Add";
+
+const invalidContents = {
+	tags: (value: string | string[]) => typeof value === "string",
+	paragraph: (value: string | string[]) => Array.isArray(value),
+	labelValue: (value: string | string[]) => Array.isArray(value),
+	file: (value: string | string[]) => Array.isArray(value),
+} 
+
+const Content = (props: ContentProps): ReactElement => {
+	const { title, onEdit, content, onAdd } = props;
+	const [show, setShow] = useState(content?.length < 3);
+
+	/**
+	 * Validates if the combinations of the type of content and
+	 * value are allowed.
+	 */
+	useEffect(() => {
+		if(!content) return;
```

Comment:
Codacy has a fix for the issue: [Expected space(s) after "if".](https://app.codacy.com/gh/simpleviewinc/sv-mosaic/pullRequest?prid=9571838)
```suggestion
		if (!content) return;
```